src/client/Components/Routes.tsx

```diff
@@ -1,22 +1,44 @@
 import React from "react";
-import { IonPhaser } from "@ion-phaser/react";
+import { Action } from "redux";
 import { connect } from "react-redux";
-import { HashRouter as Router, Route } from "react-router-dom";
-import PropTypes from "prop-types";
+import { ThunkDispatch } from "redux-thunk";
+import { IRootState as AppState } from "../store/Reducers";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { getLoginStatus } from "../store/Reducers/loginReducer/asyncActions";
+import SpotifyLogin from "./Spotify_Components/SpotifyLogin";
+import { Home } from "./Home";
 import Footer from "./Footer";
-import { game } from "./phaser/game";
 
-class Routes extends React.Component {
+interface AppProps {
+  fetchLoginStatus: () => void;
+}
+
+const mapStateToProps = ({ loggedinStatus }: AppState) => ({
+  loggedinStatus,
+});
+
+const mapDispatchToProps = (
+  dispatch: ThunkDispatch<AppState, null, Action>
+) => ({
+  fetchLoginStatus: () => dispatch(getLoginStatus()),
+});
+
+class Routes extends React.Component<AppProps & AppState> {
+  componentDidMount() {
```

Comment:
since you're using hooks, you dont need class components or componentdidmount
just use useEffect